renku-cli-model/src/test/scala/io/renku/cli/model/generators/SoftwareAgentGenerators.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.renku.cli.model.generators
+
+import io.renku.cli.model.CliSoftwareAgent
+import io.renku.graph.model.RenkuTinyTypeGenerators
+import org.scalacheck.Gen
+
+trait SoftwareAgentGenerators {
+
+  def softwareAgentGen: Gen[CliSoftwareAgent] =
+    for {
```

Comment:
Just wondering: I saw you changed many for comprehensions to start on the `def` line - but this one is on the line below (which is my personal pref 😆 so I like it hehe). Is this perhaps a written formatting rule somewhere?

Reply:
I don't think we've got a rule rewriting that anywhere. I somehow have a clashing preference so it might be that when I do some changes in a file where I see your formatting, my auto-manual formatter could kick-off. Sorry...

Reply:
No worries! I don't have a strong opinion here and might also mix it sometimes. was genuinely wondering as I looked through